test/k8sT/KafkaPolicies.go

```diff
@@ -34,8 +34,8 @@ var _ = Describe("K8sKafkaPolicyTest", func() {
 		microscopeCancel                       = func() error { return nil }
 		backgroundCancel    context.CancelFunc = func() { return }
 		backgroundError     error
-		l7Policy            = helpers.ManifestGet("kafka-sw-security-policy.yaml")
-		demoPath            = helpers.ManifestGet("kafka-sw-app.yaml")
+		l7Policy            = ""
+		demoPath            = ""
```

Comment:
Maybe comment that these are set in BeforeAll below. It's more than a page away and may not be obvious.